Loop/View Controllers/CarbAbsorptionViewController.swift

```diff
@@ -130,6 +130,7 @@ final class CarbAbsorptionViewController: LoopChartsTableViewController, Identif
             currentContext.formUnion(RefreshContext.all)
         }
         charts.startDate = chartStartDate
+        charts.updateEndDate(chartStartDate.addingTimeInterval(.hours(totalHours+1))) // allow 1 hour for presenting current data and looking into the future
```

Comment:
Maybe the comment should note that this doesn't actually end up being used when we have data; the data specifies the end date used. This is just for the no-data case.

Reply:
I could see this comment leading a dev to think they can show more future data by updating this, but it wouldn't work.